kubernetes-with-otelcol-tracing/templates/config/collector.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $.Values.baseName }}-opentelemetry-collector
+  namespace: {{ $.Values.namespace }}
+  labels:
+    app: {{ $.Release.Name }}-opentelemetry-collector
+    service: buildbarn
+data:
+  config.yaml: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+    processors:
+      batch:
+        timeout: 10s
+      k8sattributes:
+        passthrough: true
+    exporters:
+      datadog:
+        env: {{ $.Values.otelcol.env }}
+        service: {{ $.Values.namespace }}
+        tags:
+          - kube_namespace:{{ $.Values.namespace }}
+          - deployment:{{ $.Values.baseName }}
+        api:
+          key: example-key
+          site: datadoghq.com
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch, k8sattributes]
+          exporters: [datadog]
```

Comment:
This seems to be invalid?
> Error: cannot load configuration: unknown exporters type "datadog" for datadog
2021/12/28 01:08:34 collector server run finished with error: cannot load configuration: unknown exporters type "datadog" for datadog

Reply:
huh, interesting. I didn't even modify the collector config, I just copy and pasted it directly from my deployment, which is definitely working.